### PR TITLE
new vignette for infillcrits

### PR DESCRIPTION
**Pull request by verenamayer**

### CODE REVIEW

File: vignettes/Vignette_Infillcrits.Rmd [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+title: "mlrMBO: Infill criteria"
+output: html_document
```
Comment (jakob-r) [inline vignettes/Vignette_Infillcrits.Rmd]: Can you follow the header and the structure of the other vignettes please?
For running text I strongly favor a line break after each sentence.

**Comment by berndbischl** [comment]
> Did @berndbischl asked you for it or how does it come?

yes, verena had nice examples in her thesis, i asked her for help here

> I am bit afraid we will add a lot if redundancy here if we are not carefull.

sure, but lets figure it out quickly and resolve it. where is stuff redundant now?

**Comment by verenamayer** [comment]
To avoid redundancy, maybe it's better to take only the examples and put them to the end of the quick introduction?

**Comment by berndbischl** [comment]
before we avoid redundancy lets please state where it currently exists

**Comment by jakob-r** [comment]
Let's not make the quick introduction "unquicker" :wink: I am currently reading the text and shortening it a bit.

**Comment by verenamayer** [comment]
Thats right :-) I think the only redundancy is, that the general configuration is explained twice. In the quick introduction and here.

**Comment by berndbischl** [comment]
> Thats right :-) I think the only redundancy is, that the general configuration is explained twice. In the quick introduction and here

good. then lets reduce this here. a LITTLE bit of redundancy does not hurt, and can even be good. but lets not overdo this.

**Comment by berndbischl** [comment]
@jakob-r we should iterate and merge quickly

**Comment by jakob-r** [comment]
I'm fine with the state now. Can be merged.

**Pull request merged**